recipe/conda_forge_ci_setup/utils.py

```diff
@@ -9,9 +10,68 @@
 except ImportError:
     from yaml import safe_load
 
+import rattler_build_conda_compat.render
+
 CONDA_BUILD = "conda-build"
 RATTLER_BUILD = "rattler-build"
 
+
+def get_built_distribution_names_and_subdirs(recipe_dir=None, variant=None, build_tool=None):
+    feedstock_root = os.environ.get(
+        "FEEDSTOCK_ROOT",
+        os.getcwd(),
+    )
+    if recipe_dir is None:
+        recipe_dir = os.path.join(feedstock_root, "recipe")
+    if build_tool is None:
+        build_tool = determine_build_tool(feedstock_root)
+    if variant is None:
+        if "CONFIG_FILE" in os.environ:
+            variant = [os.environ.get("CONFIG_FILE")]
+        else:
+            variant = [
+                os.path.join(
+                    os.environ.get("CI_SUPPORT", os.path.join(feedstock_root, ".ci_support")),
+                    os.environ.get("CONFIG") + ".yaml"
+                )
+            ]
```

Comment:
This is fine to have, but we should deprecate these.

Reply:
Yeah it'd be nice to cleanup the set of supplied env vars. It differs by platform and possibly provider in smithy. 

Which ones did you want to deprecate?

Reply:
`recipe_dir` being None and `variant` being None. They should be passed through to this function.

Reply:
Yes, I understand why there needs to be default values for now. We should fix smithy, so that it is not needed in the future.